rx_paparazzo/src/main/java/com/miguelbcr/ui/rx_paparazzo2/interactors/TakePhoto.java

```diff
@@ -94,6 +96,16 @@ private Intent getIntentCamera(Uri uri) {
     Intent intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
     intent.putExtra(MediaStore.EXTRA_OUTPUT, uri);
 
+    if (CameraMode.FRONT.equals(config.getCameraMode())) {
+      intent.putExtra("android.intent.extras.CAMERA_FACING", android.hardware.Camera.CameraInfo.CAMERA_FACING_FRONT);
+      intent.putExtra("android.intent.extras.LENS_FACING_FRONT", 1);
+      intent.putExtra("android.intent.extra.USE_FRONT_CAMERA", true);
```

Comment:
It seems that `android.hardware.Camera.CameraInfo` is deprecated from API 21 (Build.VERSION_CODES.LOLLIPOP) -> https://developer.android.com/reference/android/hardware/Camera.CameraInfo

Instead, suggests to use `android.hardware.camera2.` -> https://developer.android.com/reference/android/hardware/camera2/package-summary

I can see that `LENS_FACING_FRONT` was added from API 21:
- https://developer.android.com/reference/android/hardware/camera2/CameraMetadata#LENS_FACING_FRONT

I wonder if you have tested it in several android versions just to make sure it works, for instance, API 18, 21, 22, 27

I'll try it too...